test/tests/functional/pbs_reservations.py

```diff
@@ -1244,6 +1242,9 @@ def test_standing_resv_with_multinode_job_array(self):
         self.server.expect(RESV, {'reserve_index': 2}, offset=60, id=rid)
         self.server.expect(JOB, {'job_state=R': 2}, count=True,
                            extend='t', id=jid)
+        self.server.expect(JOB, {'job_state': 'Q'}, id=subjid[4])
         self.server.expect(JOB, 'queue', op=UNSET, id=jid, offset=60)
+        self.server.log_match(resv_queue + ";deleted at request of pbs_server",
```

Comment:
Only one comment 
This line should come before self.server.expect(JOB, 'queue', op=UNSET, id=jid, offset=60)
Rest looks fine.